A3

```diff
@@ -0,0 +1,61 @@
+"""
+Hey, is was asked to implement the following engineering specification:
+
+Write a function that when given a tic-tac-toe board (two 
+dimensional array of strings that are either 'X', 'O', or ''), 
+return the winner of the board. If there is no winner, return None.
+
+Here is the pull request.
+"""
+
+from operator import add as list_concat
+
+def get_winner(board):
+	"""
+	Returns None if no winner. Otherwise, returns "X" or "O" as winner.
+	"""
+
+	winning_lines = get_winning_lines(board)
```

Comment:
define get_winning_lines before get_winner so that the reader is familiar with it before reading get_winner